packages/edit-site/src/components/block-editor/block-editor-provider/test/use-page-content-blocks.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { renderHook } from '@testing-library/react';
+/**
+ * WordPress dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import usePageContentBlocks from '../use-page-content-blocks';
+
+jest.mock( '@wordpress/blocks', () => {
+	return {
+		__esModule: true,
+		...jest.requireActual( '@wordpress/blocks' ),
+		createBlock( name, attributes = {}, innerBlocks = [] ) {
+			return {
+				name,
+				attributes,
+				innerBlocks,
+			};
+		},
+	};
+} );
+
+describe( 'usePageContentBlocks', () => {
+	const blocksList = [
+		createBlock( 'core/group', {}, [
+			createBlock( 'core/post-title' ),
+			createBlock( 'core/post-featured-image' ),
+			createBlock( 'core/query' ),
+			createBlock( 'core/post-content' ),
+		] ),
+		createBlock( 'core/query' ),
+		createBlock( 'core/paragraph' ),
+		createBlock( 'core/post-content' ),
+	];
+	it( 'should return empty array if `isPageContentFocused` is `false`', () => {
+		const { result } = renderHook( () =>
+			usePageContentBlocks( blocksList, false )
```

Comment:
Tiny nit: feel free to disregard!

Would it be clearer if `usePageContentBlocks` accepted an object instead of positional params? That way the boolean in the second param could be clearer as to what the value refers to.

On the other hand, since it is a boolean, it makes for a nice on / off switch irrespective of what the source of the boolean is... so I don't mind it after all. Ignore me! 😄

Reply:
Fair comment! I also thought about whether the args were appropriate but shelved it as a refactor job since it's only local to the component.

Reply:
Absolutely, I'd leave as-is since it's local as mention 👍